common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -52,7 +52,9 @@ def undo_changes_via_modal(self):
         """
         Trigger clicking event of the undo changes button in the modal.
         Wait for the undoing process to load via ajax call.
+        Before that Scroll so the button is clickable on all browsers
         """
+        self.browser.execute_script("window.scrollTo(0, 500)")
```

Comment:
I'm wondering if you can get the coordinates first, then feed those coordinates into the script?

That way if the layout changes, the tests will be resilient. Plus, that pattern can be adopted elsewhere; I had seen this issue on other tests a few months ago...